client/ayon_blender/plugins/create/create_blendScene.py

```diff
@@ -6,10 +6,11 @@
 
 
 class CreateBlendScene(plugin.BlenderCreator):
-    """Generic group of assets."""
+    """Generic .blend export writing linked datablocks"""
```

Comment:
We can either link or append .blend from BlendScene product, But the docstring appears to indicate this product only exporting assets for linking scene only.
```suggestion
    """Generic .blend export writing datablocks"""
```